data/mllib/sample_bisecting_kmeans_data.txt

```diff
@@ -0,0 +1,20 @@
+6.4,2.7,5.3,1.9
```

Comment:
Yes, we can. The reason why I added a new data file was that it was a little strange to use one for k-means for me. So, I'll remove the file and then use the sample data for k-means instead of it.